src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -307,6 +326,10 @@ public function shouldReport(Throwable $e)
      */
     protected function shouldntReport(Throwable $e)
     {
+        if ($this->deduplicateReporting && ($this->reportedExceptionMap[$e] ?? false)) {
+            return true;
+        }
```

Comment:
```suggestion
        return $this->deduplicateReporting && ($this->reportedExceptionMap[$e] ?? false);
```

Reply:
There is more code underneath this statement that needs to be executed if the condition is not true.